PatchKit/Classes/Extensions/Date+Extensions.swift

```diff
@@ -34,5 +38,9 @@ extension Date {
         dateFormatterPrint.dateFormat = format
         return dateFormatterPrint.string(from: self)
     }
+    
+    func isBetween(date date1: Date, andDate date2: Date) -> Bool {
+        return date1.compare(self).rawValue * self.compare(date2).rawValue >= 0
```

Comment:
I find this to be cleaner and easier to read/understand: `return (min(date1, date2) ... max(date1, date2)).contains(self)`

Reply:
I like that a lot more than what I copy pasted from stack overflow. Will change